### PR TITLE
tweak skiplink style

### LINKED ISSUE #1268: Skiplink non-standard
## Summary

fix to look like canada.ca

## Design Detail

image

## Unresolved questions

Is this what RCMP wants?

### PR DESCRIPTION
**Pull request by sastels**
Fixes #1268 

# Description

tweak skiplink a bit to make it look more like Canada.ca

Canada.ca:

image

ours previously:

image

this PR:

image

# Checklist:

- [x] I have looked at my code on GitHub and it all looks good (ex: no random commented out code or console.logs)
- [x] I have added and needed tests for my changes (in particular for new screens)
- [x] I have added a comment to any confusing code

### CODE REVIEW

**Comment by sastels** [comment]
forgot to do a story for this - let me try this out...

File: f2/src/components/skip-link/index.js [hunk 1]
```diff
@@ -24,7 +24,9 @@ export const SkipLink = ({ invisible, ...rest }) => (
     <A
       mt={2}
       p={2}
-      backgroundColor="black"
+      textDecoration=""
+      backgroundColor="gray.skipLink"
```
Comment (amazingphilippe) [inline f2/src/components/skip-link/index.js]: If I can suggest using: `blue.800` and rounded="none"

File: f2/src/theme/canada.js [hunk 2]
```diff
@@ -25,6 +25,7 @@ const colors = {
     700: '#373737',
     800: '#202020',
     900: '#191919',
+    skipLink: '#444444',
```
Comment (amazingphilippe) [inline f2/src/theme/canada.js]: Do we need a specific color key just for skiplinks?

Comment (sastels) [reply f2/src/theme/canada.js]: not if we're using `blue.900` 😄

**Pull request merged**